packages/rum-core/src/rumEvent.types.ts

```diff
@@ -784,6 +784,34 @@ export type RumViewEvent = CommonProperties & {
     }[]
     [k: string]: unknown
   }
+  /**
+   * Display properties
+   */
+  display?: {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Is this change on [rum-events-schema](https://github.com/DataDog/rum-events-format/blob/master/schemas/rum/_common-schema.json)?